### PR TITLE
Apprentice Progress, Finished Aika's chunk

### PR DESCRIPTION
**Pull request by MusouMajo**
Still another 9 files left in ch4, thought I should submit what I have so far for proofreading.

[crysis suit voice] Maximum Chuuni

### CODE REVIEW

File: Story/Episode 5/co_052401_02.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name01#0,"""Aika"""
-skit01#0,"""What!?<br>How is this possible, my servant!?"""
-skit01#1,"""What!?<br>How is this possible, my servant!?"""
+skit01#0,"""What?!<br>My familiar, vanquished... But how?!"""
```
Comment (SynthSy) [inline Story/Episode 5/co_052401_02.csv:2]: Make sure they're both the same.

Comment (MusouMajo) [reply Story/Episode 5/co_052401_02.csv:2]: lmao I was very confused for a moment. Cursed interrobangs...

File: Story/Episode 5/st_056120.csv [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+st_056120_0210#0,"""E-everything Aika's saying is all muddled up. There's even some stuff from Oracle mixed in there..."""
+st_056120_0220#0,"""There's signs of Aika in there, but I can sense a Dark Falz-like presence too..."""
+st_056120_0230#0,"""I do not know of her past, but there is no mistake that she is under the influence of Ephimera. Do not be deceived by her words, she is dangerous."""
+st_056120_0240#0,"""It seems as though our differences are irreconcilable. A pity that, now you will all have to perish."""
+st_056120_0250#0,"""Shut your mouth, foul traitor! You think we'll lie down and let you trample all over my country?! Think again, you're going down!"""
+st_056120_0270#0,"""...Aren't you getting a little too into this, Rita?"""
+st_056120_0280#0,"""Uh, well... It... Felt like one of those moments out of a heroic story, so I wanted to give it a try..."""
+st_056120_0290#0,"""You are the one who calls herself 'Queen' of this country, are you not? You were the one who summoned us forth, so why do you stand against us now."""
+st_056120_0300#0,"""We, the Four Infernal Lords of [Apprentice], are doing exactly as you asked, no?"""
```
Comment (SynthSy) [inline Story/Episode 5/st_056120.csv]: Now, you might shoot us for this, but we really wanted to push The `Elite Four` of Apprentice.

Comment (MusouMajo) [reply Story/Episode 5/st_056120.csv]: Another friend suggested that, and I thought it wasn't chuuni enough. Especially when the point of the scene is to be overblown and dramatic. Plus, Elite Four Apprentices sounds a little silly and runs into the same problem as before, and Elite Four of [Apprentice] sounds a little ESL. I-I worked very hard on making sure it sounded the right way, I swear!

**Pull request merged**